news/_posts/core-weekly/2022-03-14-coreweekly-10-2022.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 10 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-03-14
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 7th to Sunday 13th of March 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+[write a nice message here, or remove the "General messages" section]
```

Comment:
@NeOMakinG you approved this 😄

Reply:
As I said, I checked other things, it's not because it's automated that we shouldn't verify it, it can break